embedchain/apps/PersonApp.py

```diff
@@ -51,15 +62,9 @@ class PersonOpenSourceApp(EmbedChainPersonApp, OpenSourceApp):
     """
 
     def query(self, input_query, config: QueryConfig = None, dry_run=False):
-        self.template = Template(self.person_prompt + " " + DEFAULT_PROMPT)
-        query_config = QueryConfig(
-            template=self.template,
-        )
-        return super().query(input_query, query_config, dry_run)
+        config = self.add_person_template_to_config(DEFAULT_PROMPT, config)
```

Comment:
The implication of this (I think that's also how it is right now) is that only default prompts can be used, right? Because DEFAULT_PROMPT is passed anyways.

If that's intentional, we should either create subclass of QueryConfig that does not allow template modification or, simpler, log a warning to the user if template is set, that it will be ignored.

Reply:
I now see that I am wrong, I think we should add an inline comment for this to make it clearer.